### ISSUE TITLE
cursor animation

### ISSUE THREAD

**Issue by allyraza** [onivim/oni#987]
hi,

I have been using neovim with iterm2 for sometime, recently found oni and really like oni's interface.

I would like to ask if you could please remove the cursor animation or add a config param to turn it off, when the cursor blinks it blurs and looks really weird.

I am using windows 8.1 and running latest build of oni installer.

cheers

**Comment by CrossR** [comment]
Have you tried adding `"ui.animations.enabled": false` to your `config.js`?

There was also a bug with the animations I think that was fixed in #946, so this might be fixed in the next release.

**Comment by allyraza** [comment]
that works I think I can close this

**Comment by allyraza** [comment]
I am sorry about this I think it partially fixes the problem

**Comment by badosu** [comment]
@allyraza Can you tell us why it's a partial and not a full fix?

**Comment by bryphe** [comment]
There was actually a bug where we weren't fully disabling the animation, that was fixed in #946 (it's in master, but not in a released build). It might be worth trying that build with `ui.animations.enabled` false.

When animations are on, we set a scale transform on the cursor background, it's possible there are some aliasing issues - the fix in #946 should address that. 

We also render the 'cursor character' differently - it's in the DOM, versus the canvas, which has a separate font render path - so it is possible that in itself is causing issues. 

@allyraza , if you're up for trying out the latest master with `ui.animations.enabled` set to `false`, and can check and see if you're still seeing issues with blurry characters, that would be great!
